smart-frontend/app/src/components/uikit/Calendar/Calendar.constants.ts

```diff
@@ -0,0 +1,14 @@
+export const monthsDictionary: Record<number, string> = {
```

Comment:
There is License section in README.md, should read this secation.
Should run maven command for append license info to new files